### PR TITLE
RSDK-7531 - Add GetProperties to vision service

### PR DESCRIPTION
**Pull request by kharijarrett**
The file worth viewing here (to see the changes) is the vision.proto . Adding this as scoped here (https://docs.google.com/document/d/1P0eFy1Y5zBT91oFxz32isku3l5HJUHfdzSKYuS-MFMI/edit?usp=sharing)

 
Everything else is autogenerated.

### CODE REVIEW

File: proto/viam/service/vision/v1/vision.proto [hunk 1]
```diff
@@ -160,3 +168,16 @@ message GetObjectPointCloudsResponse {
...
+  // name of the vision service
+  string name = 1;
+  // Additional arguments to the method
+  google.protobuf.Struct extra = 99;
+}
+
+message GetPropertiesResponse {
+  bool classifications_supported = 1;
+  bool detections_supported = 2;
+  bool object_point_clouds_supported = 3;
```
Comment (njooma) [inline proto/viam/service/vision/v1/vision.proto:182]: Would be nice to have docstrings here, even though the docstrings would be super self explanatory. It's just very useful with auto-generated documentation in the SDKs

**Review by njooma — APPROVED** [review]

**Comment by bhaney** [comment]
I see that in some places, the protoc version gets downgraded to a previous version. Can you make sure your local setup is up to date ('make setup' on your api repo) and that you are basing your commit on top of the latest commit of the api repo, too?

**Comment by kharijarrett** [comment]
Added docstrings and also did the make setup (I'm on the freshest api commit). That protoc version is still downgrading though, so I feel like it's supposed to.

**Comment by kharijarrett** [comment]
Okay great catch on the versioning thing. It's fixed now, I had to nuke my old /opt folder before the make setup.

**Review by bhaney — APPROVED** [review]
Thanks for doing the checks! LGTM

**Pull request merged**